db/system_keyspace.cc

```diff
@@ -1201,6 +1204,18 @@ schema_ptr system_keyspace::view_building_coordinator_tasks() {
     return schema;
 }
 
+schema_ptr system_keyspace::built_tablet_views() {
+    static thread_local auto schema = [] {
+        auto id = generate_legacy_id(NAME, BUILT_TABLET_VIEWS);
+        return schema_builder(NAME, BUILT_TABLET_VIEWS, id)
+                .with_column("keyspace_name", utf8_type, column_kind::partition_key)
+                .with_column("view_name", utf8_type, column_kind::clustering_key)
+                .with_hash_version()
+                .build();
+    }();
```

Comment:
What happens if a user request a rebuild? The entry is removed?

Reply:
I believe that rebuild is _currently_ not supported in tablets-enabled keyspaces:

https://github.com/scylladb/scylladb/blob/88d3fc68b51209485b792ccecde5545933f9d6ac/service/storage_service.cc#L4894-L4899

The view build coordinator currently only handles tablet-enabled keyspaces.

Reply:
Indeed, the entry wasn't removed when a view was dropped. Fixed this but also I've removed this table (and commit) and now `system.built_views` (existing table) is used.